### PR TITLE
CUST-1705  responsive config app

### PR DESCRIPTION
**Pull request by DarrylD**

### CODE REVIEW

File: client/app/common/directives/page-header-options/page-header-options.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    justify-content: flex-end;
+  }
+
+  .peach-page-header-options{
+
+    .md-menu{
+      position: relative;
+    }
+
+    .md-menu:after {
```
Comment (jsmylnycky) [inline client/app/common/directives/page-header-options/page-header-options.scss]: Can you please put this after inside of the above .md-menu?  ie

.md-menu {
  position: relative;

  &:after {
    .....
  }
}

**Comment by jsmylnycky** [comment]
@DarrylD  Fix up the one thing I mentioned and then you can go ahead and merge into master.  Tag me here when its ready and I'll get it released.  Thanks!

**Comment by DarrylD** [comment]
@jsmylnycky done

**Comment by jsmylnycky** [comment]
Released v0.1.17

**Pull request merged**